docs/autotools.rst

```diff
@@ -1,5 +1,13 @@
 Building with configure (\*nix including GNU/Linux)
 ---------------------------------------------------------------------
+
+To install Universal-ctags' dependencies on Debian-based systems, do::
+
+	$ sudo apt install \
+		  pkg-config autoconf python3-docutils \
+		  libseccomp-dev libseccomp2 \
+		  libjansson-dev libjansson4
```

Comment:
I think `libyaml-dev` and `libxml2-dev` should be also added.

`libseccomp2` and `libjansson4` might not be necessary, because they will be automatically installed as the dependencies of `libseccomp-dev` and `libjansson-dev`.

Reply:
@k-takata, thank you. Obviously, you are correct. I must be more careful when reviewing.